docs/API.md

```diff
@@ -725,6 +726,10 @@ If you want all transactions of all derivation schemes of all crypto currencies,
 
 As an alternative to get notification, you can also use long polling with the [event stream](#eventStream).
 
+Fields:
+
+* `replacing`: An array of all transaction ids which have been directly or indirectly replaced by this transaction.
```

Comment:
Maybe a note here, saying something like: "This can be used to detect when an output for a tracked address is replaced by a transaction with an output *for that same address*, regardless of amount."

I'm not sure if "for that same address" is true, maybe it should say "for any tracked address" or "for any address in the same derivation scheme", but it should specify the conditions for this property being set.

Reply:
Reworded:

`The list of the unconfirmed transactions of this wallet which have been replaced by this new transaction. This can typically be used to detect when the sender is bumping fee. This can't be used to detect when the sender is attempting to abort a transaction.`